releases/release-1.24/release-team.md

```diff
@@ -5,7 +5,7 @@
 | Lead | James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) / Slack: `@james.laverack`) | Jesse Butler ([@jlbutler](https://github.com/jlbutler) / Slack: `@Jesse Butler`), Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Cici Huang ([@cici37](https://github.com/cici37) / Slack: `@cici37`), Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) |
 | Enhancements | Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`) | |
 | CI Signal | Leonard Vincent Simon Pahlke ([@leonardpahlke](https://github.com/leonardpahlke) / Slack: `@lvspahlke`) | |
-| Bug Triage | Jyoti Mahapatra ([@jyotimahapatra](https://github.com/jyotimahapatra) / Slack: `@Jyoti Mahapatra`) | |
+| Bug Triage | Jyoti Mahapatra ([@jyotimahapatra](https://github.com/jyotimahapatra) / Slack: `@Jyoti Mahapatra`) | Dipto Chakrabarty ([@DiptoChakrabarty (https://github.com/DiptoChakrabarty)] / Slack: `@Dipto`) | Heba Elayoty ([@helayoty](https://github.com/helayoty) / Slack: `@helayoty`) | Igor Andriushchenko ([@doshyt](https://github.com/doshyt) / Slack: `@Igor Andriushchenko`) | Ritu Panjwani ([@ritpanjw](https://github.com/ritpanjw) / Slack: `@ritpanjw`) |
```

Comment:
nit
There are too many pipe symbols (`|`) replace the pipe between shadows with a comma (`,`) - see section Docs below as reference

Reply:
ah, didnt notice that. Thanks!